language-server/modules/langserver-core/src/test/resources/completion/new/statement_context/config/foreach_stmt_ctx_config6.json

```diff
@@ -478,20 +478,6 @@
       "insertText": "numberList",
       "insertTextFormat": "Snippet"
     },
-    {
-      "label": "doSomeTask()",
```

Comment:
For the following cursor position

```
public function testFunction() {
    int[] numberList = [1, 2, 3];

    foreach int a in d<cursor>
}

function doSomeTask() returns int[] {
    // todo: here goes a certain task
}
```

The `doSomeTask` function is not visible.